sql/synth_recipes.sql

```diff
@@ -2337,7 +2337,7 @@ INSERT INTO `synth_recipes` VALUES (32532,0,2008,0,0,41,58,0,0,0,0,4099,4241,745
 INSERT INTO `synth_recipes` VALUES (32533,0,2008,0,0,41,58,0,0,0,0,4099,4241,745,816,1647,1647,1647,0,0,0,1968,1968,1968,1968,2,4,6,8,'Shiny Gold Thread');
 INSERT INTO `synth_recipes` VALUES (32534,0,2008,0,0,41,58,0,0,0,0,4099,4241,745,816,1642,1642,1647,0,0,0,1967,1967,1967,1967,2,4,6,8,'Dull Gold Thread');
 INSERT INTO `synth_recipes` VALUES (32535,0,0,0,0,0,58,0,0,0,0,4099,4241,1278,1278,1278,0,0,0,0,0,13273,13274,13274,13274,1,1,1,1,'Twinthread Obi');
--- INSERT INTO `synth_recipes` VALUES (32536,0,0,0,0,255,58,0,0,0,0,4099,4241,788,788,822,827,828,0,0,0,12731,12793,12793,12793,1,1,1,1,'Velvet Cuffs');
+INSERT INTO `synth_recipes` VALUES (32536,0,0,0,0,255,58,0,0,0,0,4099,4241,788,788,822,827,828,0,0,0,12731,12793,12793,12793,1,1,1,1,'Velvet Cuffs');
```

Comment:
255

Reply:
sorry about that, rushed out the door this morning and forgot to change those values... thanks for catching it